docusaurus.config.js

```diff
@@ -184,15 +184,16 @@ module.exports = {
       },
     ],
     [
-      // Camunda 8 REST API docs generation
+      // Orchestration cluster REST API docs generation
       "docusaurus-plugin-openapi-docs",
       {
         id: "api-camunda-openapi",
         docsPluginId: "default",
         config: {
           camunda: {
             specPath: "api/camunda/camunda-openapi.yaml",
-            outputDir: "docs/apis-tools/camunda-api-rest/specifications",
+            outputDir:
+              "docs/apis-tools/orchestration-cluster-api-rest/specifications",
```

Comment:
I love that this leaves everything else untouched (like GH workflows and developer guides and such)